### PR TITLE
Use Eask for CI

### PR DESCRIPTION
**Pull request by jcs090218**
This is the same PR with https://github.com/emacs-lsp/lsp-ui/pull/700 and https://github.com/emacs-lsp/dap-mode/pull/596

I think tests on Windows never work?

### CODE REVIEW

File: lsp-dart-test-support.el [hunk 1]
```diff
@@ -181,9 +181,9 @@ NOTIFICATION is the event notification.")
   "Spawn COMMAND with ARGS on a separated buffer."
   (lsp-dart-test--clean-process-buffer)
   (let ((process-buffer (get-buffer-create lsp-dart-test--process-buffer-name))
-        (project-root (lsp-dart-get-project-root)))
+        (project-rt (lsp-dart-get-project-root)))
     (with-current-buffer process-buffer
-      (setq-local default-directory (or project-root default-directory))
+      (setq-local default-directory (or project-rt default-directory))
```
Comment (jcs090218) [inline lsp-dart-test-support.el:186]: I have temporary rename the variable to `project-rt`, or else the `package-lint` will give me the following error:

```
184:9: error: You should depend on (emacs "28.1") if you need ‘project-root’.
```

**Comment by ericdallo** [comment]
Thank you for this!
I think tests on windows started to fail a while a go and I never managed to fix it :(

**Review by ericdallo — APPROVED** [review]
LMK if you think it' s ready to merge

**Comment by jcs090218** [comment]
> I think tests on windows started to fail a while a go and I never managed to fix it :(

No worries! We can fix this in the later PR. ;)

> LMK if you think it' s ready to merge

It's ready! :)

**Pull request merged**